conans/client/migrations.py

```diff
@@ -95,7 +95,7 @@ def _make_migrations(self, old_version):
             migrate_editables_use_conanfile_name(cache)
 
         if old_version < Version("1.31.0"):
-            migrate_tgz_location(self.cache, self.out)
+            migrate_tgz_location(cache, self.out)
```

Comment:
This is weird, it is already fixed in develop

Reply:
lemme force a pull here

Reply:
Okay, fixed after doing a new pull.